tests/TestCase/Shell/Task/MigrationSnapshotTaskTest.php

```diff
@@ -141,6 +175,14 @@ public function getBakeName($name)
         return $name;
     }
 
+    /**
+     * Assert that the $result matches the content of the baked file
+     *
+     * @param string $bakeName Name of the file to compare to the test
+     * @param string $result Results generated by the test to be compared
+     *
+     * @return void
```

Comment:
To be honest, I never really pay attention to that.
But if we don't use it, I should probably remove them to keep consistency.